setup/launch.py

```
@@ -34,7 +20,8 @@
     validate_environment, validate_port, validate_host
 )
 from setup.services import (
```

Flake8 (7.3.0)

[error] 'setup.services.validate_services' imported but unused

(F401)